Objective-C/ChartboostExample/ViewController.m

```diff
@@ -27,6 +28,13 @@ - (void)viewDidLoad {
     [self log:self.logBeforeViewDidLoad];
 }
 
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:animated];
+    
+    //Showing an explanation on this permission to the user prior to making the request may lead to a better user experience and improved monetization for your app.
+    [self requestPermission];
```

Comment:
Sorry just one more minor thing, we can rename this to something that indicates what this permission is?
Maybe `requestTrackingAuthorization`?

Reply:
Same in Swift

Reply:
👍